src/components/EditProfile/EditProfile.jsx

```diff
@@ -0,0 +1,144 @@
+import dynamic from "next/dynamic"
+import { useState } from "react"
+import PhoneInput from "react-phone-input-2"
+
+import "react-phone-input-2/lib/style.css"
+
+import Button from "../button"
+import Input from "../input"
+
+const Maps = dynamic(() => import("./Maps"), {
+  ssr: false,
+})
+
+export default function EditProfile() {
+  // * Form data handler
+  const [formData, setFormData] = useState({
+    name: "",
+    surname: "",
+    email: "",
+    phoneNumber: "",
+    newPassword: "",
+    confirmNewPassword: "",
+    address: "",
+  })
+
+  // * Form submitting handler
+  const handleSubmit = (event) => {
+    event.preventDefault()
+
+    // * Password comparison
+    if (formData.newPassword !== formData.confirmNewPassword) {
+      return alert("New passwords do not match.")
+    }
+
+    // * Confirmation window
+    const confirmSave = window.confirm("Save changes?")
+    if (confirmSave) {
+      // * display form values in console
+      console.log(formData)
+
+      // * save changes
+      alert("Changes saved.")
+    }
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className='grid lg:grid-cols-2 lg:ml-36 lg:w-[100%] gap-x-12 w-[60%] mx-auto'>
+        <span className=' lg:h-[448px] flex flex-col justify-between'>
+          <Input
+            name='name'
+            type='text'
+            placeholder='Name'
+            required={true}
+            minLength={2}
+            maxLength={50}
+            pattern='[a-zA-Z]+'
+            value={formData.name}
+            onChange={(e) => setFormData({ ...formData, name: e.target.value })}
```

Comment:
You can do it smarter, and use the function I made in the signup instead of writing many functions. The function called handleChange